<?xml version="1.0" encoding="UTF-8"?><testrun name="PaymentControllerTest" project="payment-service" tests="3" started="3" failures="1" errors="0" ignored="0">
  <testsuite name="com.project.controller.PaymentControllerTest" time="0.357" displayname="PaymentControllerTest" uniqueid="[engine:junit-jupiter]/[class:com.project.controller.PaymentControllerTest]">
    <testcase name="testSearchPayment" classname="com.project.controller.PaymentControllerTest" time="0.269" displayname="testSearchPayment()" uniqueid="[engine:junit-jupiter]/[class:com.project.controller.PaymentControllerTest]/[method:testSearchPayment()]">
      <failure>java.lang.AssertionError: Status expected:&lt;302&gt; but was:&lt;404&gt;&#13;
	at org.springframework.test.util.AssertionErrors.fail(AssertionErrors.java:59)&#13;
	at org.springframework.test.util.AssertionErrors.assertEquals(AssertionErrors.java:122)&#13;
	at org.springframework.test.web.servlet.result.StatusResultMatchers.lambda$matcher$9(StatusResultMatchers.java:637)&#13;
	at org.springframework.test.web.servlet.MockMvc$1.andExpect(MockMvc.java:214)&#13;
	at com.project.controller.PaymentControllerTest.testSearchPayment(PaymentControllerTest.java:74)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="testGetAllPayments" classname="com.project.controller.PaymentControllerTest" time="0.026" displayname="testGetAllPayments()" uniqueid="[engine:junit-jupiter]/[class:com.project.controller.PaymentControllerTest]/[method:testGetAllPayments()]"/>
    <testcase name="test_AddPayment" classname="com.project.controller.PaymentControllerTest" time="0.057" displayname="test_AddPayment()" uniqueid="[engine:junit-jupiter]/[class:com.project.controller.PaymentControllerTest]/[method:test_AddPayment()]"/>
  </testsuite>
</testrun>
